pool.go

```diff
@@ -109,8 +113,14 @@ func (p *Pool) startWorker(w Worker) {
 			if job == nil {
 				continue
 			}
+			if p.isClosed.Load().(bool) {
+				// update job to db
+				if err := job.Update(stores.STATUS_PENDING); err != nil {
```

Comment:
The update queries for job with id in DB and update the status, so if we don't have that job in DB, this doesn't do anything?

Reply:
It is basically rely on the implementation of the job (implementation of the JobHandler). In this case, update acting as inserting